tests/integration/spreedcheats/lib/Controller/ApiController.php

```diff
@@ -85,6 +85,15 @@ public function resetSpreed(): DataResponse {
 			))
 			->executeStatement();
 
+		try {
+			$delete = $this->db->getQueryBuilder();
+			$delete->delete('notifications')
+				->where($delete->expr()->eq('app', $delete->createNamedParameter('spreed')))
+				->executeStatement();
+		} catch (\Throwable $e) {
+			// Ignore
+		}
```

Comment:
The other queries are not guarded against exceptions, should they be guarded too or they will always succeed and this one has something special that could cause the query to throw in some cases?

Reply:
This one is guarded as it is against the table of another app and I thought better save then annoyed